packages/cactus-plugin-ledger-connector-fabric/src/main/json/openapi.json

```diff
@@ -452,9 +453,44 @@
                     "success": {
                         "type": "boolean",
                         "nullable": false
+                    },
+                    "transactionId":{
+                        "type": "string",
+                        "nullable": false
                     }
                 }
             },
+            "GetTransactionReceiptResponse": {
+                "type": "object",
+                "properties": {
+                   "blockNumber":{
+                       "type":"string"
+                   },
+                   "channelID":{
+                       "type": "string"
+                   },
+                   "transactionCreator":{},
```

Comment:
@jscode017 After having finished developing the feature, did you come across more information regarding the shape/types of these variables that could be used to narrow down the type definitions from `any`? (same question for all the other members that have no type information here at the moment.

Reply:
@jscode017 could you work on this? The types that you obtain from the transaction receipt are enough for now

Reply:
resolved